guides/user/managed-delivery/index.md

```diff
@@ -1,49 +1,43 @@
 ---
 layout: single
-title:  "Quick Reference"
+title:  "Overview"
 sidebar:
   nav: guides
-redirect_from: /reference/managed-delivery/
 ---
 
 {% include toc %}
 
-This is a quick reference for Managed Delivery. 
+> Managed Delivery is currently in Alpha and is not recommended for managing 
```

Comment:
Should we add this:

`{% include alpha version="1.18" %}`

Reply:
Does this mean "available in alpha as of 1.18" or is it something that will need to be updated with every spinnaker release to denote "still alpha"?

Reply:
The way it is now, we don't have good process. We add the alpha tag, with the as-of version, then leave it there until someone notices it's been awhile. I _think_ it's the same on the code side--alpha until someone decides _ad hoc_ that it's no longer alpha, or beta. 

@plumpy , is that the case?

Reply:
Yeah, there's nothing I know of in the code to denote which features are alpha, beta, etc. so this is just a documentation thing.